.github/workflows/pr-checks.yaml

```diff
@@ -107,14 +107,34 @@ jobs:
         with:
           names: operator-refs-${{github.event.pull_request.number}}-${{github.event.pull_request.head.sha}}
 
-      - name: Acceptance tests
+      - name: Deploy SBO
         if: ${{ steps.check-skip-acceptance.outputs.can_skip != 'true' }}
         timeout-minutes: 60
         run: |
           source ./operator.refs
           export CATSRC_NAME=sbo-pr-checks
 
-          make SKIP_REGISTRY_LOGIN=true -o registry-login test-acceptance-with-bundle
+          make SKIP_REGISTRY_LOGIN=true -o registry-login deploy-from-index-image
+
+      - name: Patch SBO to accept OLM annotations
+        if: ${{ steps.check-skip-acceptance.outputs.can_skip != 'true' }}
+        run: |
+          echo "##[group]kubectl describe deployments.apps -n operators service-binding-operator"
+            kubectl describe deployments.apps -n operators service-binding-operator
+          echo "##[endgroup]"
+          echo "##[group]Patch operator"
+            kubectl patch deployments.apps -n operators service-binding-operator --type=json -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--olm-descriptor"}]'
```

Comment:
@sadlerap are you sure this change to the deployment is not reverted back by the OLM immediatelly after it is patched?